test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -76,7 +80,7 @@ Describe "Select-String" -Tags "CI" {
         }
 
         it "Should output a string with the first match highlighted" {
-            if ($Host.UI.SupportsVirtualTerminal)
+            if ($Host.UI.SupportsVirtualTerminal -and $OutputPreference -eq 'AllowVtEscapeSequences')
```

Comment:
I think this is from a previous change

Reply:
Missed this in my search, will remove the latter part.